server/meta.jsx

```diff
@@ -74,6 +74,11 @@ function isSDKUrl(hostname : string) : boolean {
     return false;
 }
 
+function isLocalUrl(hostname : string) : boolean {
+    const localUrls = ['localhost', 'ngrok', 'loca'];
+    return localUrls.some(url => hostname.includes(url));
```

Comment:
Can't do `.includes()` since it would match something like `localhost.attacksite.com`. Would need to define an array of full domains we want to match.

Reply:
@bluepnume Provided definitive hosts.